src/sentry_core.c

```diff
@@ -206,9 +221,11 @@ sentry_capture_event(sentry_value_t event)
                     sentry__path_filename(attachment->path)));
         }
 
+        sentry__record_errors_on_current_session(1);
```

Comment:
well, `capture_event` can be anything, not necessarily an error? Not quite sure what the expectations around this are…

Reply:
Yeah. Updated based on python logic now (more or less). It does not check mechanism because presumably crash we always detect more reliably here.